integration/runner/runner.go

```diff
@@ -77,23 +86,35 @@ func RunCommand(cmd string, args ...string) error {
 	return nil
 }
 
+func RunSshCommand(cmd string, args ...string) error {
+	a := []string{}
+	if *sshConfig != "" {
+		a = append(a, "-F", *sshConfig)
+	}
+	a = append(a, args...)
+	return RunCommand(cmd, a...)
+}
+
 func PushAndRunTests(host, testDir string) error {
 	// Push binary.
 	glog.Infof("Pushing cAdvisor binary to %q...", host)
-	args := common.GetGCComputeArgs("ssh", host, "--", "mkdir", "-p", testDir)
-	err := RunCommand("gcloud", args...)
+	ipAddress, err := getIp(host)
+	if err != nil {
+		return fmt.Errorf("failed to get ip address of host: %s, %v", ipAddress, err)
+	}
+
+	err = RunSshCommand("ssh", host, "--", "mkdir", "-p", testDir)
```

Comment:
Alternatively, go has an [ssh client](https://godoc.org/golang.org/x/crypto/ssh). The advantage to using the go client is you wouldn't need to open and close multiple connections, so it seems like it would be less flaky? Although if you weren't using multiple connections, the connection would be open for longer, so I'm not sure which would be better.